OPT_mission.Altis/warehouse/functions/fnc_spawnvehicle.sqf

```diff
@@ -105,7 +105,12 @@ for "_i" from 0 to _spiralMaxPoints step 1 do
 		// update budget initialized by server!
 		[UNIT_NAME(_unit), UNIT_SIDE(_unit), _vecType, _unitCost, "-"] call EFUNC(common,updateBudget);
 		[] remoteExecCall [QFUNC(updateBudget), _unit, false];
-
+		
+		if ((_vec isKindOf "OPT_B_Heli_Light_01_F") or (_vec isKindOf "OPT_O_Heli_Light_01_F")) then 
```

Comment:
Sind die Flares nicht im Mod raus? Oder muss das in beidem (Mod und Mission) eingestellt werden?

Reply:
kann raus, hat im mod keine Flares und müssen hier nicht mehr entfernt werden.